test cases/common/48 test args/meson.build

```diff
@@ -27,9 +27,5 @@ test('file arg', testerpy, args : testfile, env : env_array)
 
 copy = find_program('copyfile.py')
 tester = executable('tester', 'tester.c')
-testfilect = custom_target('testfile',
-                           input : testfile,
-                           output : 'outfile.txt',
-                           build_by_default : true,
-                           command : [copy, '@INPUT@', '@OUTPUT@'])
+subdir('sub1')
```

Comment:
If this is a test requirement to have testfile inside subdir then add a comment explaining why. Otherwise someone will rearrange it later and this will be forgotten. Alternatively (better) consider asserting that there is at least one `/` in path in `copyfile.py`.

Reply:
I wanted the test to be in a subdir so that there's at least one path separator in `@OUTPUT@`. Will add a comment.